enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/handlers/DuplexPipelineWrapperFactory.java

```diff
@@ -19,13 +19,14 @@
  */
 package org.neo4j.causalclustering.handlers;
 
+import org.neo4j.graphdb.config.Setting;
 import org.neo4j.kernel.configuration.Config;
 import org.neo4j.kernel.impl.util.Dependencies;
 import org.neo4j.logging.LogProvider;
 
 public interface DuplexPipelineWrapperFactory
```

Comment:
This interface was already a bit weird, but is becoming even weirder now that we supply it both Config and a specific Setting. Not crucial for this PR, but just mentioning it.